tests/api/moderator-onboarding.test.ts

```diff
@@ -111,12 +112,26 @@ describe("moderator-onboarding", () => {
         },
       });
 
-      expect(mockEmails.send).toHaveBeenCalledWith({
-        to: "testing@gmail.com",
-        subject: "Sign Up For PR - Good Dog Licensing",
-        html: `<p>Follow <a href="${getBaseUrl()}/pnr-invite/?id=${moderatorInvite?.moderatorInviteId}">this link</a> to sign up as a PR.`,
-        from: env.VERIFICATION_FROM_EMAIL ?? "",
-      });
+      const emailParams = new EmailParams()
+        .setFrom(
+          new Sender(
+            "message@test-p7kx4xwv6peg9yjr.mlsender.net",
```

Comment:
Again, should use env variable here:

```suggestion
           env.GOOD_DOG_FROM_EMAIL ?? "",
```